.github/workflows/build_rocks.yaml

```diff
@@ -25,10 +25,14 @@ on:
         type: string
         description: The working directory for jobs
         default: "./"
-      cache-action:
+      rockcraft-ref:
+        description: Used in conjunction with rockcraft_repo to pull and build rockcraft from source instead of using snapstore version.
```

Comment:
```suggestion
description: Used in conjunction with rockcraft-repository to pull and build rockcraft from source instead of using snapstore version.
```
just using the full input var name so it's less confusing :)

Reply:
Updated, thanks!